### DISCUSSION TITLE
Sumatra PDF

### DISCUSSION THREAD

**Discussion by Malcolm-Hood** [sumatrapdfreader/sumatrapdf#3838] [Documentation]
I like Sumatra PDF but why when I open a PDF does it seemingly cycle through every PDF I've ever opened and what's with all the xxxxxxxxxx

**Comment by GitHubRulesOK** [comment]
you have defaults set to remember opened files so the full tabs bar has all filenames with just the close button x showing.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/569996d4-e251-43be-8011-8dcb73b9bd60)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/640f3e31-741f-430f-81a6-25098250c1f9)

**Reply by Malcolm-Hood** [reply]
Many thanks for that quick fix.
